docs/content/operating.html.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Operating Flynn
+layout: docs
+toc_min_level: 2
+---
+
+# Operating Flynn
+
+We've designed Flynn to be very easy to get up and running with quickly.
+However, there are a variety of things that should be considered as you start
+using Flynn more.
+
+## Cluster Requirements
+
+At least three hosts are required to deploy Flynn in a highly available
+configuration. We do not recommend single-node clusters for production use.
+A three-host cluster will withstand the loss of one node with little to no
+impact on availability.
+
+All members of the initial cluster participate in Raft consensus voting, hosts
+started after the initial bootstrap act as proxies to the consensus cluster. We
+recommend starting with three or five hosts and adding more hosts when
+necessary.
+
+Each host should have a minimum of 1GB of memory, and inter-host network packets
+should have a latency of less than 2ms. Deploying a single Flynn cluster across
+higher latency WAN links is not recommended, as it can have a negative impact on
+cluster consensus.
+
+## Storage
+
+Flynn uses ZFS to store data. By default, a ZFS pool is created in a sparse file
+on the existing filesystem. This is not recommended for production for
```

Comment:
I have seen some users have `/var/lib/flynn` on a dedicated disk, so is it worth clarifying what "existing filesystem" means? For example:

```
By default, a ZFS pool is created in a sparse file on the
same filesystem that `/var/lib/flynn/volumes` exists on.
```